astroplan/observer.py

```diff
@@ -1663,3 +1663,35 @@ def target_hour_angle(self, time, target):
             hour_angle = Longitude(self.local_sidereal_time(time) - coord.ra)
 
         return hour_angle
+
+    @u.quantity_input(horizon=u.deg)
+    def tonight(self, horizon=0 * u.degree):
+        """
+        Return a time range corresponding to the nearest night
+
+        This will return a range of `Time` values from the `astronomical_twilight`
```

Comment:
You need to add the full namespace for the functions/classes to make sphinx recognize it. e.g. `~astropy.time.Time`. The ``~`` in the front means that the path will be hidden in the rendered documentation. 
(Need to do it for ``astronomical_twilight`` and ``astronomical_sunrise``, too)